src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+
+    return title.toLowerCase().includes(query.toLowerCase().trim())
+      || description.toLowerCase().includes(query.toLowerCase().trim());
```

Comment:
You may have created a separate varible for query.toLowerCase().trim() to not repeat yourself.